src/test/java/it/gov/pagopa/admissibility/BaseIntegrationTest.java

```diff
@@ -93,6 +93,11 @@
                 "app.pdv.retry.delay-millis=5000",
                 "app.pdv.retry.max-attempts=3",
                 //endregion
+
+                //region delete
+                "app.delete.paginationSize=100",
+                "app.delete.delayTime=1000"
+                //end region
```

Comment:
You could use this values directly as default in the application.yml: there is not reason to change them here